### PR TITLE
Updated versions and fixed build for 1.16.3

### PR DESCRIPTION
**Pull request by Tankonor**
Fixed the build issues and updated dependency versions.

### CODE REVIEW

**Comment by Tankonor** [comment]
Should clarify, this is for 1.16.3

**Comment by KnightMiner** [comment]
Can you explain the reason for this PR? Yes, we can update versions, but why? It builds just fine with the current versions, and being built against the current versions still allows it to run with the latest Forge.

My general policy on version changes in PRs is only do it if needed for another feature (e.g. a PR needs a feature from the latest Forge) or if part of a substantial update (e.g., porting to 1.16.4).

File: gradle.properties [hunk 1]
```diff
@@ -14,15 +14,15 @@ minecraft_version_max=1.17
...
 
 # Mappings Information
 mappings_version=20200916-1.16.2
 
 # Build Dependencies
 mantle_version=1.6.38
-jei_version=7.1.+
-hwyla_version=1.10.10-B77
-probe_version=3.0.2-beta-5
+jei_version=7.5.0.43
```
Comment (AlphonseSantoro) [inline gradle.properties]: Restricting versions to a specific one is never a good idea for compatibility issues, unless you have a very good reason then this should be `7.5.+`

**Comment by Tankonor** [comment]
> Can you explain the reason for this PR? Yes, we can update versions, but why? It builds just fine with the current versions, and being built against the current versions still allows it to run with the latest Forge.
> 
> My general policy on version changes in PRs is only do it if needed for another feature (e.g. a PR needs a feature from the latest Forge) or if part of a substantial update (e.g., porting to 1.16.4).

It seems the last PR into master broke the build (project is not currently building) so I wanted to fix it.

Build ref:
https://dvs1.progwml6.com/jenkins/blue/organizations/jenkins/SlimeKnights%2FTinkersConstruct/detail/1.16/19/pipeline

File: gradle.properties (hunk 1 above)
Comment (Tankonor) [reply gradle.properties]: No problem, will correct this!

**Comment by KnightMiner** [comment]
Build did not fail because of build.gradle, it randomly failed to connect to all Maven repos, versions had nothing to do with it.

Restarted the build and 20 worked: https://dvs1.progwml6.com/jenkins/blue/organizations/jenkins/SlimeKnights%2FTinkersConstruct/detail/1.16/20/pipeline

Since the build worked fine, I'm going to close this. As I said, in general we don't take PRs that just update build numbers, the time it takes to review it is much more than it would take to just do it ourselves, so only update build numbers in a PR if its relevant to the actual PR content.

**Pull request closed without merging**